### ISSUE TITLE
Accessibility: [Screen Reader]:  Selecting spacebar/enter when the screen reader focus is on container response options, always selects the first response

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3084]
![image](https://user-images.githubusercontent.com/1005609/78628310-a3d75c80-7848-11ea-8e9f-07364016905e.png)

**Actual**
With JAWS ON and keyboard tab navigation, notice that the screen reader focus comes to “response container”. JAWS announces all the responses/answers one after the other and then says “press spacebar to select”. When user enters spacebar always the first option is selected.  (see screenshot below –Are you experiencing any of the following symptoms ?]
 
**Expected**
JAWS focus should not come to “response container” during keyboard tab navigation. The selection experience is completely unpredictable. 
 
**WCAG Violation**
Name, Role Value: 4.1.2
https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html

Info and Relationships 1.3.1
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

**Comment by compulim** [comment]
This is a bug on Adaptive Cards.

The content author is creating those buttons using hero card, which is rendered through Adaptive Cards.

@corinagum can you find the AC bug and associate this with it?

**Comment by compulim** [comment]
I believe this is because the AC card container is focusable, when it shouldn't be.

In our offline chat, we refer to this phenomena as "rogue tabindex=0". Will file a bug to AC and link it here.
